lit/docs/guides/git.lit

```diff
@@ -0,0 +1,141 @@
+\title{Git Guides}
+
+\use-plugin{concourse-docs}
+\split-sections
+\table-of-contents
+
+\section{
+  \title{Basic Git Operations}
+  \omit-children-from-table-of-contents
+
+  All of these examples use the
+  \link{concourse/git-resource}{https://github.com/concourse/git-resource}
+  image. That image is probably the most popular git resource for Concourse
+  since it is shipped in the
+  \link{concourse/concourse}{https://hub.docker.com/r/concourse/concourse}
+  image and in the tarball on the \link{Github release
+  page}{https://github.com/concourse/concourse/releases}. It is not the only
+  resource available for working with git-related resources. If you don't see
+  your use-case on this page then there is probably another resource that you
+  can use. For example, Pull Request workflows can be accomplished with the
+  \link{teliaoss/github-pr-resource}{https://github.com/telia-oss/github-pr-resource}.
+
+  Check out the \link{docs}{https://github.com/concourse/git-resource} for the
+  git resource for all configuration options.
+
+  \section{
+    \title{Fetching a Repository}
+
+    Here is how you fetch the contents of a git repository and use it in a
+    task.
+
+    \codeblock{yaml}{{
+    resources:
+    - name: concourse-examples
+      type: git
+      icon: github
+      source:
+        uri: https://github.com/concourse/examples
+
+    jobs:
+    - name: read-the-readme
+      plan:
+      - get: concourse-examples
+      - task: cat-readme
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: gitea/gitea # use any image that has the git cli
```

Comment:
Perhaps this is a little misleading, as the image used in this task doesn't need `git`